public/js/dom.js

```diff
@@ -389,29 +389,50 @@ export function displaySimulationResults_simpleMode(
     simulationWrapperDiv.scrollIntoView({ block: 'end' })
 }
 
-function getNoiseBadgeType(noiseValue) {
-    if (noiseValue >= 100) {
-        return 'over-100'
-    } else if (noiseValue >= 20) {
-        return 'over-20'
-    } else if (noiseValue >= 15) {
-        return 'over-15'
-    } else if (noiseValue >= 10) {
-        return 'over-10'
-    } else if (noiseValue >= 5) {
-        return 'over-5'
-    } else if (noiseValue >= 1) {
-        return 'over-1'
-    } else {
-        return 'under-1'
+function getNoiseBadgeType(noiseValue, percentage) {
```

Comment:
Change request: Rename to `isPercentage` to stick to conventions for booleans

Reply:
Done